### PR TITLE
Improve general styling and change the colour scheme to be a little easier on the eyes

### PR DESCRIPTION
**Pull request by erbridge**
image

### CODE REVIEW

**Comment by leeky** [comment]
@erbridge Personally I find the bright teal sidebar a little harder to read than the current dark green version, but it could be because I'm viewing it on calibrated screens 🤷

**Comment by CristinaRO** [comment]
@erbridge Yeah, I'm sorry, it's the teal in large amounts on the screen that is overwhelming, and coupled with orange or yellow it only makes it harder.

I would try blue and yellow myself if I had a spare moment to play with the dev tools: https://v-df7640e636dddb2c9710.brand.madebyfieldwork.com/section/core-elements#colour-palette

**Comment by erbridge** [comment]
I'll switch to another. This is pretty easy now.

**Comment by erbridge** [comment]
How's this?

image

**Comment by CristinaRO** [comment]
Screenshot 2020-06-19 at 12 53 24
or (my favourite so far)
Screenshot 2020-06-19 at 12 57 11
or even
Screenshot 2020-06-19 at 13 02 15

**Comment by erbridge** [comment]
I've been trying to use one that has a square avatar set up.

**Comment by erbridge** [comment]
I like the cream one actually. Let me try that.

**Comment by erbridge** [comment]
Ok, let's try again:

image

**Comment by gazaston** [comment]
> Ok, let's try again:
> 
> image

Have we contrast-checked this?

**Comment by erbridge** [comment]
Yup. Passed Firefox's contrast check.

**Comment by erbridge** [comment]
300 is a very light font-weight though for Poppins.

**Comment by erbridge** [comment]
I've made to 400 again, even though it disagrees with the brand guide.

image

**Comment by gazaston** [comment]
Screenshot 2020-06-19 at 14 19 47

Did FF think this was okay, or perhaps it doesn't check placeholders?

**Comment by erbridge** [comment]
It did. I've upped the contrast on it.

image

File: _layouts/default.html [hunk 1]
```diff
@@ -21,21 +21,6 @@
             aria-label="Search text"
             placeholder="Type to search"
           />
-          <div class="clear-button">
```
Comment (CristinaRO) [inline _layouts/default.html:24]: I would like to understand better why we're removing the clear search button without a replacement. I don't fully understand what overflowing the layout means.

Is it a functionality that can "just" be done without?

Comment (erbridge) [reply _layouts/default.html:24]: We can add it back in, but it causes the sidebar to overflow currently, and I'm tired of fighting Docsify's defaults.

Comment (erbridge) [reply _layouts/default.html:24]: And I don't think it provides much value.

**Pull request merged**